test/helpers.js

```diff
@@ -115,13 +119,15 @@ export const paginationTest = ({ validToken, invalidToken, endpoint, model, mode
         });
     });
   });
+};
 
+export const itReturnsAnErrorWhenServiceFails = (token, endpoint, model, modelMethod) => {
```

Comment:
We can set a default here for modelMethod since it is setup for 'aggregate' to ensure that we can omit the last parameter when we use this function for aggregate.

```suggestion
export const itReturnsAnErrorWhenServiceFails = (token, endpoint, model, modelMethod = "aggregate")
```